maat-court-data-api/src/main/java/gov/uk/courtdata/dto/RepOrderStateDTO.java

```diff
@@ -0,0 +1,43 @@
+package gov.uk.courtdata.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class RepOrderStateDTO {
+
+    private Integer usn;
+    private Integer maatRef;
+    private String caseId;
+    private String caseType;
+
+    private String iojResult;
+    private AssessorDetails iojAssesorName;
+    private LocalDate dateAppCreated;
```

Comment:
I've seen this typo (assesor) in a couple of places across our services, so obviously tricky to change elsewhere - however I wonder if we could correct it here because this is a new class?